js/ui/tree_view/ui.tree_view.base.js

```diff
@@ -29,6 +29,7 @@ var WIDGET_CLASS = "dx-treeview",
     NODE_CLASS = "dx-treeview-node",
     ITEM_CLASS = "dx-treeview-item",
     ITEM_WITH_CHECKBOX_CLASS = "dx-treeview-item-with-checkbox",
+    ITEM_WITHOUT_CHECKBOX_CLASS = "dx-treeview-item-without-checkbox",
```

Comment:
Hmm, I see that it class attaches onto a TreeView's node. I thinks we should rename it to avoid confusing styles like
```
.dx-treeview-item-with-checkbox {
    .dx-treeview-item { ... }
}
```

And one more thing - can we use class names without "with\without"?
Like as `dx-treeview-node-simple', 'dx-treeview-node-checkbox'?

Reply:
You are right but from another side, I think we should not rename CSS classes without a serious reason. And "without" class name is an expected pair for "dx-treeview-item-with-checkbox"

Reply:
@jsdmitry  @mrjohn0011 What's your opinion?

Reply:
Of course, names of CSS classes need to be renamed and sometimes moved to other elements but we didn't rename `dx-treeview-toggle-item-visibility` class because of potential BC. In my opinion, we can make this BC in 19.1 major release. This should be a separated task in 19.1: "Clean less in DevExtreme widgets". This is a large feature with BC in the less.

Reply:
I think that should not make BC, as many customers are may use this in the custom themes.